docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -179,6 +179,8 @@ to configure in Spring Cloud Azure 4.0, it can leverage the credential stored in
 managed identity in Azure Services, just make sure the principal has been granted sufficient permission to access the
 target Azure resources.
 
+NOTE: When assign roles to the security principals to interact with Azure messaging services, the `Data Owner` role is required to send / receive messages. For Azure Spring Cloud Stream Event Hubs / Service Bus Binder libraries, `Owner` role is required when the function of auto creating resources is needed.
```

Comment:
Which role is bigger? `Contributor` or `Owner`?

Reply:
Owner has bigger permission and we should use Contributor here.